src/index.js

```diff
@@ -97,14 +97,18 @@ class DSHttpAdapter {
   getPath(method, resourceConfig, id, options) {
     let _this = this;
     options = options || {};
-    let args = [
-      options.basePath || _this.defaults.basePath || resourceConfig.basePath,
-      this.getEndpoint(resourceConfig, (isString(id) || isNumber(id) || method === 'create') ? id : null, options)
-    ];
-    if (method === 'find' || method === 'update' || method === 'destroy') {
-      args.push(id);
+    if (isString(options.urlPath)) {
+      return makePath.apply(DSUtils, [options.basePath || _this.defaults.basePath || resourceConfig.basePath].concat(options.urlPath.split('/')));
```

Comment:
I don't think you have to split `options.urlPath`, I think you can just make it the second item in the array passed to `madePath.apply`.